README.md

```diff
@@ -3,7 +3,7 @@
 [![Docker Repository on Quay](https://quay.io/repository/prometheus/alertmanager/status "Docker Repository on Quay")][quay]
 [![Docker Pulls](https://img.shields.io/docker/pulls/prom/alertmanager.svg?maxAge=604800)][hub]
 
-The Alertmanager handles alerts sent by client applications such as the Prometheus server. It takes care of deduplicating, grouping, and routing them to the correct receiver integrations such as email, PagerDuty, or OpsGenie. It also takes care of silencing and inhibition of alerts.
+The Alertmanager handles alerts sent by client applications such as the Prometheus server. It takes care of deduplicating, grouping, and routing them to the correct [receiver integrations](https://prometheus.io/docs/operating/integrations) such as email, PagerDuty, OpsGenie, VictorOps or Zenduty. It also takes care of silencing and inhibition of alerts.
```

Comment:
nit: I think it's unfair that we list some and not others. I think the original list of email, pagerduty and Opsgenie seemed fine.

Reply:
@roidelapluie what do you think?

Reply:
I understand where you're coming from but if we start doing this, it will be a never-ending list of requests to add more. I'd agree with @gotjosh to leave the list unchanged for now.

Reply:
I understand. I guess just having a link to integrations would be a good change regardless as you mentioned earlier so can probably stick to **just** that.